ui/app/components/tabs/tab/tab.component.js

```diff
@@ -3,13 +3,13 @@ import PropTypes from 'prop-types'
 import classnames from 'classnames'
 
 const Tab = props => {
-  const { name, onClick, isActive, tabIndex } = props
+  const { name, onClick, isActive, tabIndex, className, activeClassName } = props
 
   return (
     <li
       className={classnames(
-        'tab',
-        isActive && 'tab--active',
+        className || 'tab',
+        isActive && (activeClassName || 'tab--active'),
```

Comment:
I think this might be better expressed as `{[activeClassName]: isActive}` (assuming we don't need the default as per the comment above). Thoughts?